src/js/components/Nav/Nav.stories.tsx

```diff
@@ -52,10 +52,217 @@ stories.add("Default", () => (
 stories.add("Default", () => (
   <div
     className="u-flex"
-    style={{ backgroundColor: "#FFEFA5", minHeight: "800px" }}
+    style={{ backgroundColor: "#FFEFA5", minHeight: "800px", position: "relative" }}
```

Comment:
Is this position relative needed? If so, we should find a better way to express this without requiring that its set in a style block. Maybe the nav component itself starts with a div that just sets the position to relative if thats actually required.

Reply:
Removed the need for relative positioning,.